web/channels/user_socket.ex

```diff
@@ -5,7 +5,8 @@ defmodule ColaboraApi.UserSocket do
   # channel "rooms:*", ColaboraApi.RoomChannel
 
   ## Transports
-  transport :websocket, Phoenix.Transports.WebSocket
+  transport :websocket, Phoenix.Transports.WebSocket,
+    timeout: 45_000
   # transport :longpoll, Phoenix.Transports.LongPoll
```

Comment:
Setting the timeout is required for pushing to Heroku?  They don't like instantaneous websocket connections?